src/main/resources/application-sqs.properties

```diff
@@ -1,3 +1 @@
-audit.redrive.policy={"maxReceiveCount": "${audit.queue.maximumRedeliveries}", "deadLetterTargetArn":"arn:aws:sqs:${aws.sqs.region}:${aws.account.id}:${audit.queue.dlq.name}"}
-audit.queue=aws-sqs://arn:aws:sqs:${aws.sqs.region}:${aws.account.id}:${audit.queue.name}?amazonSQSClient=#sqsClient&messageAttributeNames=All&redrivePolicy=${audit.redrive.policy}&waitTimeSeconds=20&backoffIdleThreshold=1&backoffMultiplier=${audit.queue.backOffMultiplier}&initialDelay=5000&delay=${poll.delay}
-audit.dlq=aws-sqs://arn:aws:sqs:${aws.sqs.region}:${aws.account.id}:${audit.dlq.name}?amazonSQSClient=#sqsClient&messageAttributeNames=All
\ No newline at end of file
+aws.queue.audit.url=aws-sqs://arn:aws:sqs:${aws.sqs.region}:${aws.account.id}:${aws.queue.audit.name}
```

Comment:
Does this work? this is still camel notation. it should be 
https://sqs.<<region>>.amazonaws.com/<<accountId>>/<<queuename>> I think we can just pass in sqs_url from the k8s cs-*-audit-sqs secret